static/css/webrtc.css

```diff
@@ -50,19 +50,21 @@
   margin-right: 6px;
   cursor: pointer;
   transition: opacity .2s, background-color .2s;
-  color: #333;
+  color: rgba(51, 51, 51, 1);
```

Comment:
Meant to be as close to exactly the same as before. I just wanted to be able to make an alpha .5 version of the same below, and I figured it was more clear what was going on if they were both `rgba`.